doc/en/user/source/geowebcache/webadmin/defaults.rst

```diff
@@ -31,6 +31,11 @@ With direct integration, tile caching is enabled for all standard WMS requests t
 
 This setting is disabled by default. When enabling this option, it is a good idea to also turn on :ref:`gwc_webadmin_diskquotas` as well, to prevent unbounded growth of the stored tiles.
 
+Explicitly require TILED Parameter

```

Comment:
Nice to see docs! Maybe an update to the screenshot above would help too.

Reply:
I have updated that image